Model/AdyenThreeDS2Process.php

```diff
@@ -37,10 +37,21 @@ class AdyenThreeDS2Process implements AdyenThreeDS2ProcessInterface
      */
     private $adyenHelper;
 
+
+    /**
+     * @var \Magento\Sales\Model\OrderFactory
+     */
+    protected $orderFactory;
+
+    /**
+     * @var
+     */
+    protected $order;
+
     /**
      * @var \Adyen\Payment\Logger\AdyenLogger
      */
-    private $adyenLogger;
+    protected $adyenLogger;
```

Comment:
Is there a reason for making this more open?

Reply:
not used so will remove this.

Reply:
no will change it back.

Reply:
will make all of them private.